pkg/utils/image_test.go

```diff
@@ -40,12 +40,15 @@ var _ = Describe("Image retrieval tests", func() {
 	It("etcd spec defines etcd and backup-restore images", func() {
 		etcd = testutils.EtcdBuilderWithDefaults(etcdName, namespace).Build()
 		imageVector = createImageVector(true, true, false, false)
-		etcdImage, etcdBackupRestoreImage, err := GetEtcdImages(etcd, imageVector, false)
+		etcdImage, etcdBackupRestoreImage, alpine, err := GetEtcdImages(etcd, imageVector, false)
```

Comment:
can rename `alpine` to `initContainerImage` which is what you have called in the main code as well. Also make that change in all other places as well.

Reply:
Done